docs/examples/general/data_loading/external_input.ipynb

```diff
@@ -95,7 +95,7 @@
    "outputs": [],
```

Comment:
<p>to <strong>use the GPU version of ExternalSource operator, removing the </strong>decoder as we assume that ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Done